src/components/DeprecatedBanner/DeprecatedBanner.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import styled from 'styled-components'
+import { ModalConsumer } from '../ModalManager/ModalManager'
+import { theme, Text, SafeLink, Button } from '@aragon/ui'
+import Banner from '../Banner/Banner'
+import Modal from '../Modal/Modal'
+
+const { negative, negativeText, accent, textDimmed } = theme
+
+const DeprecatedBanner = props => (
+  <ModalConsumer>
+    {({ showModal }) => <DeprecatedDao showModal={showModal} {...props} />}
+  </ModalConsumer>
+)
+
+class DeprecatedDao extends React.Component {
+  handleClick = () => {
+    const { dao, showModal } = this.props
+    showModal(DeprecatedModal, { dao })
+  }
+
+  render() {
+    const { children, dao } = this.props
+
+    return (
+      <React.Fragment>
+        <Banner
+          text="This Rinkeby DAO will be deprecated on 1/11/18"
```

Comment:
I'd change this text to be `Old 0.5 Rinkeby DAOs will be deprecated on Nov. 1, 2018` since this appears during onboarding as well:

![image](https://user-images.githubusercontent.com/4166642/46488745-e0656a80-c803-11e8-97a4-7fb4d064f41b.png)